library/src/main/java/com/applandeo/materialcalendarview/utils/AppearanceUtils.kt

```diff
@@ -0,0 +1,70 @@
+@file:JvmName("AppearanceUtils")
+
+package com.applandeo.materialcalendarview.utils
+
+import android.graphics.drawable.Drawable
+import android.view.View
+import android.widget.TextView
+import com.applandeo.materialcalendarview.R
+import kotlinx.android.synthetic.main.calendar_view.view.*
+
+/**
+ * Created by Mateusz Kornakiewicz on 04.01.2018.
+ */
+
+fun View.setAbbreviationsLabels(color: Int, firstDayOfWeek: Int) {
+    val labels = mutableListOf<TextView>(
+            mondayLabel,
+            tuesdayLabel,
+            wednesdayLabel,
+            thursdayLabel,
+            fridayLabel,
+            saturdayLabel,
+            sundayLabel)
+
+    val abbreviations = this.context.resources.getStringArray(R.array.material_calendar_day_abbreviations_array)
+
+    (0..6).forEach {
```

Comment:
foreach with labels: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/for-each-indexed.html